### DISCUSSION TITLE
Downloads incomplete

### DISCUSSION THREAD

**Discussion by VideoUser67** [aclap-dev/video-downloadhelper#270] [Help and Q&A]
### Your question

I'm using Version 8.2.1.1 in Firefox 8.2.1.1. Recently most of my downloads fail to complete. I don't get an error; they appear to have completed but are a fraction of the actual video length, sometimes only a couple of seconds.

### Addon & CoApp version and details

Version 8.2.1.1 in Firefox 8.2.1.1

**Comment by paulrouget** [comment]
I need a URL to try to reproduce.

Also, see if the beta version fixes your issue: https://www.downloadhelper.net/beta

**Reply by VideoUser67** [reply]
Hi, Paul. This one failed twice, but then succeeded, then failed again. It's pretty hit and miss.
https://movieshd.watch/watch-tv/watch-the-walking-dead-the-ones-who-live-online-106516.10340695

**Reply by Mattmjs** [reply]
That link worked no problem, 100% complete file. But you haven't provided the complete picture here.
What's your companion app version ?  What's your OS ?  Have you checked your anti virus & firewall is not blocking the software.

**Reply by VideoUser67** [reply]
companion app: VdhCoApp 2.0.19
Windows 11
AV / FW not blocking. Sometimes the links work, sometimes they don't.

**Comment by VideoUser67** [comment]
Any update on this?

**Reply by Mattmjs** [reply]
You could try the beta extension for Firefox and see if it's the same or you get a complete file.
